code/modules/reagents/chems/chems_ethanol.dm

```diff
@@ -99,6 +89,22 @@
 	if(halluci)
 		M.adjust_hallucination(halluci, halluci)
 
+// Somewhat a dummy type for 'pure ethanol' to avoid having to set dirtiness/heating products/etc on literally everything else.
+/decl/material/liquid/ethanol/spirits
```

Comment:
not super sure about calling it 'spirits' generally. aurora has iirc `/decl/reagent/alcohol/ethanol` which could be better, since it'd open the door for having e.g. rubbing alcohol etc. in the future, or alien alcohols that don't affect humans/aliens the same, etc.? not a big deal though